mesh/amqp.c

```diff
@@ -524,11 +524,8 @@ bool mesh_amqp_set_exchange(struct mesh_amqp *amqp, const char *exchange)
 {
 	struct message *msg;
 
-	if (!exchange)
-		return false;
-
 	l_free(amqp->exchange);
-	amqp->exchange = l_strdup(exchange);
+	amqp->exchange = l_strdup(exchange ?: "not_configured");
```

Comment:
I'd rather leave this as empty and avoid configuring exchanges/queues in such cases.